__tests__/integration/one-app.spec.js

```diff
@@ -932,13 +933,269 @@ describe('Tests that require Docker setup', () => {
             // subset of registration
             waiting: null,
             installing: null,
-            active: {
+            active: expect.objectContaining({
               scriptURL: scriptUrl.replace(appAtTestUrls.fetchUrl, appAtTestUrls.browserUrl),
-              state: 'activated',
-            },
+            }),
             scope: `${appAtTestUrls.browserUrl}/`,
             updateViaCache: 'none',
           });
+          await expect(browser.executeAsync(
+            // eslint-disable-next-line prefer-arrow-callback
+            function getCacheKeys(done) { caches.keys().then(done); })
+          ).resolves.toEqual([]);
+        });
+      });
+
+      describe('caching', () => {
+        const oneAppVersionRegExp = /\/([^/]+)(?:\/i18n)?\/[^/]*\.js$/;
+
+        /* eslint-disable implicit-arrow-linebreak */
+        function _getCacheEntries(done) {
```

Comment:
As discussed, add a `// TODO` here to refactor later.

Reply:
updated dd34146